app/internal/event.py

```diff
@@ -9,46 +9,56 @@
 
 from app.database.models import Event
 
-ZOOM_REGEX = re.compile(r'https://.*?\.zoom.us/[a-z]/.[^.,\b\s]+')
+ZOOM_REGEX = re.compile(r"https://.*?\.zoom.us/[a-z]/.[^.,\b\s]+")
 
 
 def raise_if_zoom_link_invalid(vc_link):
     if ZOOM_REGEX.search(vc_link) is None:
-        raise HTTPException(status_code=HTTP_400_BAD_REQUEST,
-                            detail="VC type with no valid zoom link")
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST,
+            detail="VC type with no valid zoom link",
+        )
 
 
 def get_invited_emails(invited_from_form: str) -> List[str]:
     invited_emails = []
-    for invited_email in invited_from_form.split(','):
-        invited_email = invited_email.strip()
-        try:
-            validate_email(invited_email, check_deliverability=False)
-        except EmailSyntaxError:
-            logging.exception(f'{invited_email} is not a valid email address')
-            continue
-        invited_emails.append(invited_email)
+    if not invited_from_form:
+        invited_emails.append("")
+    else:
+        for invited_email in invited_from_form.split(","):
+            invited_email = invited_email.strip()
+            try:
+                validate_email(invited_email, check_deliverability=False)
+            except EmailSyntaxError:
+                logging.exception(
+                    f"{invited_email} is not a valid email address",
+                )
+                continue
```

Comment:
Probably shouldn't use continue or break.

Reply:
Not my code.
I added the if  not invited from form as this was producing an error for empty "invitees" field

Reply:
(so I am not sure what the effect will be, need to see who wrote this originally)

Reply:
changed this a bit. to guard clause as Yam asked + instead of continue used - try, except, else